apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -13,6 +25,13 @@
     return;
   }
 
+	// Workaround to force the mozMobileMessage in shim context use window one.
+  // Need to be removed once all the event listeners are ready for shim. 
+  if (location.pathname.indexOf('shim_host') > -1) {
+    navigator.mozMobileMessage = parent.navigator.mozMobileMessage;
+    return;
+  }
```

Comment:
maybe easier to have this in a separate JS file that we'd load only in the shim html file ?